src/utils/constants.ts

```diff
@@ -29,12 +29,21 @@ const chains = [
   },
 ]
 
+enum configureTabs {
+  general = 'general',
+  auth = 'auth',
+  store = 'store',
+  access = 'access',
+  wallet = 'wallet',
+}
+
```

Comment:
Split this into two: The type of tab and an object with tab type and label. This will help with the prop definition and simplify the markup.

```suggestion
type ConfigureTab = 'general' | 'auth' | 'store' | 'access' | 'wallet'

const CONFIGURE_TABS = [{ type: 'general', label: 'General' }, ... ]
```

Reply:
Done